.tool-versions

```diff
@@ -1 +1 @@
-ruby 3.2.2
+ruby 3.2.3
```

Comment:
It might not matter, but I had started off using the latest stable ruby (3.3.0) and when I did a deploy to Render I ran into an issue. I can't recall what it was, but something blew up when it was trying to auto-install that version of ruby. I found some documentation ([I think this](https://docs.render.com/ruby-version) that explained their default ruby pre-installed is 3.2.2 so I switched us to this version in order to avoid the Render install issue.

If you need to bump this minor version then we should just be sure that Render still works (and you may need to merge in to determine that)

Reply:
That doc also makes reference to a ruby-version so does Render need us to have that? I'd expect it can read from .tool-versions but flagging to keep an eye out for any indication that Render freaks out w/o that file

Reply:
Oh good call, I'll look into that. If nothing else maybe we can deploy the container to Render instead of relying on their notion of buildpacks...that way there's less diversity to account for in the hosting systems.